### ISSUE TITLE
Dark colours for gtk theme (matching the gnome-shell theme)

### ISSUE THREAD

**Issue by vaskark** [Magog64/SURU-PLUS#11]
Hi. I love this theme and update from git almost daily. I've just discovered that you provide gnome-shell/gtk themes along with the icons. I notice, however, that the gnome-shell theme is dark (and beautiful!) and would like to have the gtk theme match it. Is there a way to do this myself? Or will there ever be a full dark option in the future?

Thanks. And great work.

**Comment by gusbemacbe** [comment]
HI @vaskark 

1. Go to your_name/.local/share/themes (if the folder themes does not exist, just create the folder, naming it as `themes`)
2. Create another folder into this folder, and name it as "Suru++ United".
3. Open Tweaks Tool and apply "Suru++ United". 

We have already a dark icons theme and also have another repository of desaturated-blue icon themes, but we didn't make a dark GTK theme yet. They are Suru++ Dark and Suru++ Telinkrin.

**Comment by gusbemacbe** [comment]
@Magog64 

Do you know programming in CSS, HTML, R and XML for redesigning the dark GTK?
Sai programmare in CSS, HTML, R e XML per ridisegnare il GTK scuro?

**Comment by gusbemacbe** [comment]
I come to inform that @Magog64 communicated that he does not work with programming to make GTK themes. I will assume the responsibility for redesigning a dark theme.

**Comment by vaskark** [comment]
Hey! Thanks for responding.
I kinda have a setup close to that. I have a symlink in ~/.themes that points to SURU-PLUS/theme, with the link named Suru++. All my compiled apps exist in the same folder so it was just easier that way.
I guess I'll just have to wait. The current gnome-shell colour would look amazing for the gtk theme.

Keep up the great work, and thanks for the heads up.

**Comment by gusbemacbe** [comment]
You are welcome. If I do not have classes these days, I must test the dark GTK theme.

**Comment by vaskark** [comment]
Hi again. I just noticed that the gnome-shell & gtk themes are gone. Are they being moved to their own project repo by any chance?

**Comment by gusbemacbe** [comment]
Yes, I moved the folders fonts, theme and wallpapers to new repositories because they caused errors in the Suru++ Icons theme when opening Gedit via terminal, as the user has reported at OpenDesktop. I fixed `index.theme` to remove the incompatibility with Gnome related to fonts and wallpapers, cursor error because Andrea did not add `cursor.theme` to the current repository. After finishing my work with the new repositories, I will update to Github and to PPA (yes, I am working with PPA now). 

But I ask you a favour.

But if you want us to main the fonts, theme and wallpapers, I will test it with the old version of Suru++ with new `index.theme`.

**Comment by gusbemacbe** [comment]
It worked now. We do not need to remove some folders as the `index.theme` is fixed. But as for PPA, the folders must be in other repositories.

**Comment by vaskark** [comment]
Sweet. If there's a PPA is it open to the public yet?

**Comment by gusbemacbe** [comment]
No, it is still in development, as Suru++ Dark GTK theme is also still in development. When the PPA is ready, we will release the PPA to the public. 

I have just updated the repository. You can update it now.

**Comment by vaskark** [comment]
Awesome. Thanks.

**Comment by gusbemacbe** [comment]
@vaskark 

Due to @paullinuxthemer's suggestions, in the reference https://github.com/gusbemacbe/suru-plus-telinkrin/issues/1, we decided the separate the icons themes, GTK themes, wallpapers for new repositories. 

But I have a good news.

I officialised and released the PPA, yo can make a cleaning of `~/.icons` and `/local/share/*` (icons, themes, wallpapers) and add my PPA:

```
sudo add-apt-repository ppa:gusbemacbe/ppa
sudo apt-get update
sudo apt install suru-plus-pack
```

Tell me if it works for you

Observe that Suru++ GTK Dark United is still in development.

**Comment by vaskark** [comment]
PPA works great on this end!

**Comment by gusbemacbe** [comment]
Ah, I am just uploading a new version now. Wait for 30 minutes.

**Comment by vaskark** [comment]
PPA works great. Except it creates a series of folders at /:

/'Suru++
/'Suru++'
/.
/Dark
/Dark'
/Lubuntu'
/Telinkrin'
/United'
/Walls'

**Comment by gusbemacbe** [comment]
Thank you for the report. I will fix it now.

**Comment by gusbemacbe** [comment]
@vaskark, the PPAś new version is released. Remove, update and upgrade the package and tell me if it is fixed.

**Comment by vaskark** [comment]
Hi again. PPA installs fine, but I'm still getting those folders in my / location.

**Comment by gusbemacbe** [comment]
Please screenshot the location where Suru++ folders were installed because I do not know which location you were referring.

**Comment by vaskark** [comment]
[Screenshot](https://i.imgur.com/cIvqnsc.png)

These folders are referenced in the ppa suru-plus pack.

**Comment by gusbemacbe** [comment]
Ah, it is in the folder root. I will analyse the files `install`, `dirs ` and `rules `of Debian compiler.

**Comment by gusbemacbe** [comment]
HI @vaskark !


I inform you that [Suru++ Dark](/gusbemacbe/suru-plus-dark) has been upgraded and renamed to Suru++ 25 Dark. Check the screenshots: https://www.opendesktop.org/p/1245808/.
